src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -827,4 +827,10 @@
     <value>Reset tab color</value>
     <comment>Text used to identify the reset button</comment>
   </data>
+    <data name="MoveTabtoNewWindowText" xml:space="preserve">
+    <value>Move Tab to New Window</value>
+  </data>
+  <data name="MoveTabtoNewWindowToolTip" xml:space="preserve">
+    <value>Moves tab to a new window using the active profile in the current directory</value>

```

Comment:
```suggestion
  <data name="MoveTabToNewWindowToolTip" xml:space="preserve">
    <value>Moves tab to a new window</value>
```

1. Capitalize the `To` to make the bot happy
2. I'd suggest removing the "using the active profile in the current directory" - that would seemingly imply that a _new_ tab is created, when really, the current tab is just moving to the new window, connection and all.

Reply:
I don't get point 2:
"using the active profile in the current directory."
 I interpreted this to mean that the current tab information (active profile in current directory) is used for the tab you want to move.  And so you don't create a new tab, but move the current tab (using the active profile in the current directory) to the new window.